### PR TITLE
[Adjustments][Christopher] Revamped About-us

### PR DESCRIPTION
**Pull request by ChristopherCastro69**
- Revamped the about us page UI 
- Added icons for the experts field. 
- Added Slideshow animation on the picture for dynamic experience
image
image

https://github.com/user-attachments/assets/6f0b5cbb-640a-48ea-a082-bd9c8b59503d

### CODE REVIEW

File: PizzaPyWebApp/templates/partials/landing/about_us.html [hunk 1]
```diff
@@ -1,49 +1,91 @@
...
+        >
+          Get To Know Us
+        </a>
+      </div>
     </div>
+  </div>
 </div>
-
 <script>
-    document.addEventListener('DOMContentLoaded', function () {
```
Comment (cristoph143) [inline PizzaPyWebApp/templates/partials/landing/about_us.html:30]: Consider integrating this approach, as it ensures flexibility by using JSON to store data, simplifying formatting and enabling seamless updates, such as adding new information.

https://github.com/pizzapy/pizzapy-website/blob/main/PizzaPyWebApp/static/data/lang.json

Comment (cristoph143) [reply PizzaPyWebApp/templates/partials/landing/about_us.html:30]: @ChristopherCastro69 💯 👍🏻

Comment (jackjk521) [reply PizzaPyWebApp/templates/partials/landing/about_us.html:30]: Minor comment on the padding on the left and right portion of the about us component, increase it by a bit

Overall looks good to me

Comment (jackjk521) [reply PizzaPyWebApp/templates/partials/landing/about_us.html:30]: @ChristopherCastro69

Comment (ChristopherCastro69) [reply PizzaPyWebApp/templates/partials/landing/about_us.html:30]: > Consider integrating this approach, as it ensures flexibility by using JSON to store data, simplifying formatting and enabling seamless updates, such as adding new information.
> 
> https://github.com/pizzapy/pizzapy-website/blob/main/PizzaPyWebApp/static/data/lang.json

hala oo sakto sd
gegege

Comment (ChristopherCastro69) [reply PizzaPyWebApp/templates/partials/landing/about_us.html:30]: image
Resolved conflicts boss @jackjk521 @cristoph143

**Review by jackjk521 — APPROVED** [review]
LGTM

**Pull request merged**